app/models/v2/policy_host.rb

```diff
@@ -8,5 +8,13 @@ class PolicyHost < ApplicationRecord
 
     belongs_to :policy, class_name: 'V2::Policy'
     belongs_to :host, class_name: '::Host'
+
+    class << self
+      def import_from_policy!(policy_id, host_ids)
```

Comment:
Shorter if you only have a single method:
```suggestion
    def self.import_from_policy!(policy_id, host_ids)
```